### PR TITLE
Feature/alert harvester documentation

### PR DESCRIPTION
**Pull request by mgdaily**
Add example for writing a custom TOM Alert Broker.

### CODE REVIEW

File: docs/create_broker.md [hunk 1]
```diff
@@ -0,0 +1,246 @@
...
+Begin by creating a file `my_broker.py`, and placing it in the `mytom/` directory
+of the project. `my_broker.py` will contain the classes that define our custom
+TOM Alert Broker.
+
+Our custom broker relies on the TOM Toolkit modules that were installed in the
+[Getting Started](/docs/getting_started) guide. Begin by editing `my_broker.py`
+to import the necessary modules.
+
+```python
+from tom_alerts.alerts import GenericQueryForm, GenericAlert, get_service_class
```
Comment (Fingel) [inline docs/create_broker.md]: We shouldn't need the get_service_class import, that is internal.

File: docs/create_broker.md [hunk 2]
```diff
@@ -0,0 +1,246 @@
...
+        score=alert['score']
+    )
+```
+In our case, the `GenericAlert` attributes match up *almost* directly with our test
+data. How convenient! We'll just go ahead and define the `GenericAlert`'s `url`
+field as the `broker_url` we retrieved our test data from.
+
+```python
+@classmethod
+def to_generic_alert(clazz, alert):
```
Comment (Fingel) [inline docs/create_broker.md]: I think the above example to_generic_alert is sufficient, no need to repeat it here.

File: docs/create_broker.md [hunk 3]
```diff
@@ -0,0 +1,246 @@
...
+./manage.py migrate
+./manage.py runserver
+```
+
+Navigate to [http://127.0.0.1:8000/alerts/query/list/](http://127.0.0.1:8000/alerts/query/list/)
+
+You should now see 'MyBroker' listed as a broker! Clicking the link will bring you
+to the query page, where you can make a query to our sample dataset.
+
+![successful-broker-list]({{"/assets/img/create_broker_doc/success_broker_list.png" | absolute_url}})
```
Comment (Fingel) [inline docs/create_broker.md]: These images don't display for me. But I don't think you need to use a special tag here. 

`successful-broker-list`

Worked fine for me.

**Review by Fingel — CHANGES_REQUESTED** [review]
A few changes

**Comment by mgdaily** [comment]
@Fingel Thanks for catching those. Got 'em cleaned up.

**Review by Fingel — APPROVED** [review]

**Comment by Fingel** [comment]
Looks great! Merge whenever you want.

**Pull request merged**